javascript/javascript1/week1/homework/ageCalculator.js

```diff
@@ -0,0 +1,22 @@
+console.log("====Human Age Checker====")
+/* human age calculator */
+const yearOfBirth = 1994;
+const yearFuture = 2027;
+
+const age = yearFuture - yearOfBirth;
+console.log("You will be ", age, "years old in ", yearFuture);
+
+/* dog age calculator */
+console.log("===dog Age Checker===")
+const dogYearOfBirth= 2035
+const dogYearFuture = 2045
+let dogYear = dogYearFuture - dogYearOfBirth
+const shouldShowresultInDogsYear = true
+if(shouldShowresultInDogsYear){
+    dogYear *= 7
+    console.log("Your dog will be",dogYear,"dog years old in", dogYearFuture)
```

Comment:
Same as the previous comment. This can be made prettier. If not with template literals, you could use the ` + " " + ` thing.
```suggestion
    console.log("Your dog will be " + dogYear + " dog years old in " + dogYearFuture);
```